### DISCUSSION TITLE
Batched toolio evaluation

### DISCUSSION THREAD

**Discussion by chimezie** [OoriData/Toolio#40] [Ideas]
This [enhancement](https://github.com/ml-explore/mlx-lm/issues/566) should facilitate this nicely via batched_generation w/ logits processors.
